setup/network.py

```
@@ -0,0 +1,165 @@
+            "addr",
+            "add",
+            f"{conf.dhcp.server_ip}/{str(conf.dhcp.subnet).split("/")[-1]}",
+            "dev",
+            conf.ap.iface,
+        ],
+        as_sudo=True,
+        log=verbose,
+        retcodes=(0, 2),
+    )
+    #
+    with open(f"/tmp/meep.ag", "w") as file:
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)